src/dynamics/video_player/player/player.js

```diff
@@ -628,6 +628,9 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         this.__setPlayerEngagement();
                     },
                     "next:resetNextWidget": function() {
+                        if (Info.isSafari()) {
+                            this._renderVideoFrame(this.__video, true);
+                        }
```

Comment:
Hi @maxblessed, why are we rendering on `resetNextWidget`? Those two things aren't related, so it might be better if we render it under some other condition